### PR TITLE
A Game of Towers - Curing Shrine Fix

### PR DESCRIPTION
**Pull request by Oroxzy**
Makes Curing Shrine usable again after 0.5 seconds (from TC bruteforced GameObjects)

### CODE REVIEW

File: sql/migrations/20220228181818_world.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20220228181818');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20220228181818');
+-- Add your query below.
+
+-- Makes Curing Shrine usable again after 0.5 seconds (from TC bruteforced GameObjects)
+UPDATE `gameobject_template` SET `data3`=500 WHERE `entry`=181955;
+UPDATE `gameobject_template` SET `data3`=500 WHERE `entry`=181682;
```
Comment (ratkosrb) [inline sql/migrations/20220228181818_world.sql:13]: Is the gobject not there in mass parse or wdb parses?

Comment (Oroxzy) [reply sql/migrations/20220228181818_world.sql:13]: it is, with the same value as before.

Comment (ratkosrb) [reply sql/migrations/20220228181818_world.sql:13]: So you are replacing the classic data with mists of pandaria data from tc?

**Comment by pjaszko** [comment]
image
how i can fix this?

**Pull request closed without merging**